lib/liquid/block_body.rb

```diff
@@ -92,7 +92,7 @@ def parse(tokenizer, parse_context, &block)
           end
           parse_context.trim_whitespace = false
           @nodelist << token
-          @blank &&= !!(token =~ WhitespaceOrNothing)
+          @blank                      &&= !!(token =~ WhitespaceOrNothing)
```

Comment:
This is the side-effect of using automated corrections.. :disappointed: :
As a human, you can tell from a distance that this change significantly hampers readability instead of improving it..
(View the entire file for proper judgement).

Reply:
This is a good way to be able to find where needing to space should be done. Searching for 15+ spaces provides most of the weird auto-corrections in which case I would space